projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -805,6 +848,50 @@ export class OwcJsonService {
 
   /** Misc --------------------------------------------------- */
 
+  private getMinMaxZoom(resource: IOwsResource, targetProjection: string = 'EPSG:4326'): { minZoom: number; maxZoom: number; } {
+    const zooms = {minZoom: 0, maxZoom: 12};
+    if (resource.properties.minZoom) {
+      zooms.minZoom = resource.properties.minZoom;
+    } else if (resource.properties.maxscaledenominator) {  // *Max*ScaleDenom ~ *Min*Zoom
+      zooms.minZoom = this.scaleDenominatorToZoom(resource.properties.maxscaledenominator, targetProjection) || 0;
+    }
+    if (resource.properties.maxZoom) {
+      zooms.maxZoom = resource.properties.maxZoom;
+    }  else if (resource.properties.minscaledenominator) {  // *Min*ScaleDenom ~ *Max*Zoom
+      zooms.maxZoom = this.scaleDenominatorToZoom(resource.properties.minscaledenominator, targetProjection) || 12;
+    }
+    return zooms;
+  }
+
+  /**
+   * Based on the WMS Standard (https://portal.ogc.org/files/?artifact_id=14416),
+   * to which the OWC Standard refers for the scale-denominator-field,
+   * and the way that openlayers calculates zoom and resolution
+   * (https://openlayers.org/en/latest/doc/tutorials/concepts.html)
```

Comment:
That might be helpful too https://wiki.openstreetmap.org/wiki/MinScaleDenominator